### PR TITLE
Add more tests.

### PR DESCRIPTION
**Pull request by jdalton**
This PR gets the ball rolling on more test additions.
I expect some tweaking of the additions to be more explicit as per snover guidance.

### CODE REVIEW

**Comment by jdalton** [comment]
I expect some tweaking of the additions to be more explicit as per guidance.

File: tests/functional/_pointerevent_pointerleave_does_not_bubble-manual.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findByXpath('id("instructions")')
+					.moveMouseTo(90, 21.859375)
+					.clickMouseButton(0)
+					.end()
+				.findByXpath('id("complete-notice")')
+					.moveMouseTo(97, 29.859375)
+					.clickMouseButton(0)
```
Comment (bethge) [inline tests/functional/_pointerevent_pointerleave_does_not_bubble-manual.js]: How comes you do mouse clicks in this test and couldn't you use `#target0` and `#parent0` directly?

**Comment by scottgonzalez** [comment]
@jdalton ping

File: tests/functional/_pointerevent_pointerleave_does_not_bubble-manual.js (hunk 1 above)
Comment (jdalton) [inline tests/functional/_pointerevent_pointerleave_does_not_bubble-manual.js]: I'm guessing it was required to get the test to work. If you think it's tweakable and not major the PR could be merged and then things tweaked/modified to taste after.

Comment (scottgonzalez) [inline tests/functional/_pointerevent_pointerleave_does_not_bubble-manual.js]: This test isn't even being run. If I do let it run, it fails.

**Pull request closed without merging**